install.sh

```diff
@@ -777,9 +779,17 @@ get_free_range() {
 set_user_subuids_subgids() {
 
   local min_range_size=65536
-  local current_user=$(get_user)
+  local current_user="${OPT_USER:-$(get_user)}"
```

Comment:
Stick to the current style and pass parameters explicitly to functions instead of using globals like `OPT_...`.